### DISCUSSION TITLE
What does it mean Prune Errored  Messages?

### DISCUSSION THREAD

**Discussion by daniel-ls-howard** [nextgenhealthcare/connect#5909] [Q&A] [answered]
It seems this would prune messages even if they had errors but right below that is the statement: "incomplete, errored, and queued messages will not be pruned.".

So, are these pruned? 

TIA

**Comment by jonbartels** [comment, marked as answer]
[The doco about pruning is not very helpful](https://docs.nextgen.com/bundle/Mirth_User_Guide_41/page/connect/connect/topics/c_Message_Pruning_Settings_connect_ug.html)

Messages that are incomplete or queued are still being processed. Pruning those would be very bad because the messages would be deleted before being processed!

Errored messages are not pruned by default because the assumption is that if there is an errored message you want to keep it to reprocess it or fix it. There is a setting in a recent MC release to allow errored messages to be pruned.
